buildenv/jenkins/JenkinsfileBase

```diff
@@ -1664,4 +1668,33 @@ def forceCleanWS() {
 	}
 }
 
-return this
+def checkDacapoH2Metric() {
+  def regex = ".*=====\\sDaCapo.*?h2\\sPASSED\\sin\\s(\\d*\\.?\\d*)\\smsec\\s=====\\s*\$"
```

Comment:
nitpicking: Please use the same indentation as the rest of the file. I think tab is used (not space)